conf/messages

```diff
@@ -1762,37 +1762,38 @@ responsiblepeople.country.of.birth.heading = Was {0} born in the UK?
+tcsp.whatyouneed.requiredinfo.text.2 = if you create complex corporate structures for clients
+tcsp.whatyouneed.requiredinfo.text.3 = if you use the services of another trust or company service provider
+tcsp.whatyouneed.requiredinfo.text.4 = your trust or company service provider’s Money Laundering Regulations number
 
-tcsp.kind.of.service.provider.title = What kind of Trust or Company Service Provider are you?
-tcsp.service.provider.lbl.01 = Nominee shareholders provider
-tcsp.service.provider.lbl.02 = Trustee provider
+tcsp.kind.of.service.provider.title = What type of trust or company service provider are you?
+tcsp.service.provider.lbl.01 = Company director, secretary, or partner provider
+tcsp.service.provider.lbl.02 = Nominee shareholders provider
 tcsp.service.provider.lbl.03 = Registered office, business address, or virtual office services provider
-tcsp.service.provider.lbl.04 = Company director, secretary, or partner provider
+tcsp.service.provider.lbl.04 = Trustee provider
 tcsp.service.provider.lbl.05 = Trust or company formation agent
 tcsp.off-the-shelf.companies.lbl = Do you sell only off-the-shelf companies?
 tcsp.create.complex.corporate.structures.lbl = Do you create complex corporate structures for clients, or are you planning to do so in the future?
-tcsp.servicesOfAnotherTcsp.title = Does your business use the services of another Trust or Company Service Provider?
+tcsp.servicesOfAnotherTcsp.title = Does your business use the services of another trust or company service provider?
 tcsp.servicesOfAnotherTcsp.other.hint = Only include companies which are based in the UK and supervised by HM Revenue and Customs for Money Laundering Regulations.
-tcsp.servicesOfAnotherTcsp.mlrregno.lbl = What’s that company’s Money Laundering Regulations registration number?
+tcsp.servicesOfAnotherTcsp.mlrregno.lbl = Money Laundering Regulations number.
 tcsp.cya.mlrregno = Money Laundering Regulation reference number
 
 tcsp.provided_services.title = Which services does your business provide?
-tcsp.provided_services.service.lbl.01 = Phonecall handling
+tcsp.provided_services.service.lbl.01 = Conference room
 tcsp.provided_services.service.lbl.02 = Email handling
 tcsp.provided_services.service.lbl.03 = Email server
-tcsp.provided_services.service.lbl.04 = Self-collect mailboxes
-tcsp.provided_services.service.lbl.05 = Mail forwarding
-tcsp.provided_services.service.lbl.06 = Receptionist
-tcsp.provided_services.service.lbl.07 = Conference rooms
+tcsp.provided_services.service.lbl.04 = Mail forwarding
+tcsp.provided_services.service.lbl.05 = Self-collect mailboxes
+tcsp.provided_services.service.lbl.06 = Telephone call handling
+tcsp.provided_services.service.lbl.07 = Receptionist
 tcsp.provided_services.service.lbl.08 = Other
-tcsp.provided_services.service.lbl.details = Please give details
+tcsp.provided_services.service.lbl.details = Service type
```

Comment:
Above you have changed the labels... so now user will select for example 'Conference room' but this will be translated to 'Phone handling' anyway because model is constructed like that... so user submitting application with 'Conference room' will eventually get back the 'Phone handling'. So it cannot be done like that I think...

Reply:
Sorted :+1: